### ISSUE TITLE
the call oci_database_db_system needs to expose the db_unique_name for a DBCS system

### ISSUE THREAD

**Issue by eugsim1** [oracle/terraform-provider-oci#1413]
Hello team
The terraform API doesnt exposes the db_unique_name which should be available as an option when the db is created.
there is a discrepancy from the REST API check the go code here :
 https://docs.oracle.com/en-us/iaas/api/#/en/database/20160918/DbSystem/LaunchDbSystem

DbUniqueName: common.String("EXAMPLE-dbUniqueName-Value"),

the missing  attribute in the TF API is present on the the REST /go calls.
in other words is the users provides a db_unique_name take it otherwise calculate it from the db_name
this is a normal operation for any oracle  databases !

Can you please upgrade ASAP the call ?
thanks
Eugene

**Comment by ravinitp** [comment]
We are very sorry that we couldn't respond to each and every issue reported on GitHub. 
Although we have refined the process to prioritize customer issues on GitHub, since this issue was reported a while ago, there is a good chance it may have been fixed in the latest version of Terraform Provider OCI. 
If you are still experiencing this issue, please create a new issue and label it as Bug.
